### PR TITLE
Analysis with InaSAFE v4

### LINKED ISSUE #334: run version 4 analysis
child of #53

### PR DESCRIPTION
**Pull request by myarjunar**
fix #334 
needs #336

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -251,20 +251,12 @@ def get_default_impact_title(self):
...
     def impact_function_name(self):
-        for i in self.impact_function_list():
-            if i['id'] == self.impact_function_id:
-                return i['name']
-        return ''
+        # Set impact function name from provenance data
+        from geosafe.tasks.headless.analysis import get_keywords
+        from geosafe.helpers.utils import get_layer_path
+        keywords = get_keywords.delay(get_layer_path(self.impact_layer)).get()
+        return keywords['provenance_data']['impact_function_name']
```
Comment (lucernae) [inline models.py:259]: I don't think we still need this?
What is it for?

**Review by lucernae — APPROVED** [review]
Is this a partial PR?
In that case, I think you can merge it as long as it doesn't broke GeoNode (because we don't have unittest yet.

File: models.py (hunk 1 above)
Comment (myarjunar) [reply models.py:259]: it's only for a view in analysis-list, I think it's still fine to show the name of the IF there

**Comment by myarjunar** [comment]
@lucernae ok thanks, we can merge it after #336

**Pull request merged**